webroot/css/sentences/search.css

```diff
@@ -16,6 +16,31 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+@media only screen and (max-width: 960px) {
+    .search_bar {
+        display: none;
+    }
+
+    #search-bar {
+        display: flex;
+    }
```

Comment:
I think there is room for improvement in the naming of these class and id. :smile: 
How about: search-bar-full and search-bar-shrinkable?
It looks like both could be made ids.